blaze/utils.py

```diff
@@ -310,3 +310,13 @@ def __get__(self, instance, owner):
             return self
 
         return self._f(instance)
+
+
+class BlazeDeprecationWarning(DeprecationWarning):
+    """Blaze specific deprecation warning."""
+    pass
+
+
+class BlazePendingDeprecationWarning(PendingDeprecationWarning):
```

Comment:
As discussed, let's remove this subclass and just use the `BlazeDeprecationWarning` here.